meeting_scheduler/meetingscheduler/src/main/java/com/spring/rest/meetingscheduler/repository/EmployeeRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.spring.rest.meetingscheduler.repository;
+
+import com.spring.rest.meetingscheduler.entity.Employee;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface EmployeeRepository extends JpaRepository<Employee, Integer> {
+    Employee findByEmployeeId(int id);
```

Comment:
id should be maintained in long format research why and provide pointers

Reply:
Long is used for ID's since it provides a larger range when compared to int. Also since it is auto incremented, if the DB grows int cannot withstand the range of the values.